config/docker-ecr-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this is used by the ECR credentials helper to make authenticating to ECR inside of a codebuild job easier.